test/unit_tests/arkode/C_serial/ark_test_erkstepsetforcing.c

```diff
@@ -0,0 +1,386 @@
+/* -----------------------------------------------------------------------------
```

Comment:
I'm concerned about how much code is duplicated here with the corresponding arkstep version. Now that forcing is define at the ARKode level, I think we can have one test file that covers several steppers. I realize that's easier said than done and that it's probably not feasible to do now.

Reply:
I'll give this some thought, since we might also want to eventually add a unit test for mristepsetforcing.  However, I think that for now these can remain separate.